### ISSUE TITLE
The VLA accuracy differs between a Jetson AGX Orin and a workstation running the same model.

### ISSUE THREAD

**Issue by shifsa** [NVIDIA/Isaac-GR00T#712]
Hello,

I want to run inference at the edge using a model trained on a workstation equipped with an RTX 6000 Ada GPU (GR00T N1.7).
I have an NVIDIA Jetson AGX Orin 64GB, and I completed the setup on bare metal following the official instructions.

When running inference in PyTorch (without TensorRT optimization), the task accuracy was lower compared to the workstation.
We expect similar inference accuracy between the workstation and the Jetson when using the same model and dataset, but observe a consistent degradation on Jetson.

We directly transferred the pre-trained model (fine-tuned on the workstation) to the Jetson.
There is also a difference in performance between the workstation and the Jetson AGX Orin even under open-loop inference.

What could be causing this discrepancy?

scripts/deployment/standalone_inference_script.py --model-path 〇〇 --dataset-path 〇〇 --embodiment-tag NEW_EMBODIMENT --traj-ids 0 1 2 --inference-mode pytorch --action-horizon 8
Workstation
INFO:root:  Average MSE across all trajs: 0.001195
INFO:root:  Average MAE across all trajs: 0.009696
Jetson
INFO:root:  Average MSE across all trajs: 0.003043
INFO:root:  Average MAE across all trajs: 0.013828

- Workstation environment:
  - Ubuntu: 22.04
  - GPU: RTX 6000 Ada
  - CUDA: 13.0

- Jetson environment:
  - Device: AGX Orin 64GB
  - JetPack: 6.2.2
  
Thank you!
